EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001267	I am so sorry, but can you get me Holy Water from the Priest Master in Klaipeda?{nl}If we have enough holy water, we'll be able to purify the demons' fallen souls.
+QUEST_LV_0300_20160310_001268	Please use the statue of Goddess Vakarine near Viyme Fountain.{nl}You can quickly come back from Klaipeda.
+QUEST_LV_0300_20160310_001269	With this, you'll be able to purify the souls contained within the Soul Sphere.{nl}Now, give me a moment.
+QUEST_LV_0300_20160310_001270	Since the power of a demon's soul is strong, the residual power after purification possesses strength as well.{nl}This power has the capability to destroy the sinister idol.{nl}
+QUEST_LV_0300_20160310_001271	Put the purified Soul Sphere's power into the sinister idol.{nl}I hope the idol can't endure this...
+QUEST_LV_0300_20160310_001272	Finally!{nl}Now, we can be freed from the black wall.{nl}
+QUEST_LV_0300_20160310_001273	It's been such a long time...{nl}I think my prayer to the goddess was not in vain...
+QUEST_LV_0300_20160310_001274	Now, we'll need to focus on the main culprit of the Tree of Truth sect, Dominikas...{nl}Everything will be over once we drive him out.{nl}
+QUEST_LV_0300_20160310_001275	Please lend me your power a little longer.{nl}Even if we have the methods to catch him, we can't use our powers {nl}since the villagers have been possessed from being tortured so long...{nl}
+QUEST_LV_0300_20160310_001276	He seems to have some power to control the plants freely.{nl}I have to warn you because he is not to be underestimated.
+QUEST_LV_0300_20160310_001277	Dominikas... He ran away...{nl}
+QUEST_LV_0300_20160310_001278	We shouldn't worry too much.{nl}It's a big harvest that we drove out the Tree of Truth sect from Elgos Monastery.{nl}
 QUEST_LV_0300_20160310_001279	And the monsters instead of plants' explosion, I guess it stopped since the spell was unstable.{nl}We were able to protect the villagers because of you.{nl} 
 QUEST_LV_0300_20160310_001280	When we get better, we will clean up the monastery.{nl}Of course, we should drive out the demons.{nl}
-QUEST_LV_0300_20160310_001281	Even if the world collapses down, we have hope like you..{nl}You give us the power to endure.{nl}
-QUEST_LV_0300_20160310_001282	I don't want to imagine what would have occurred without you..{nl}I really appreciate you on behalf of the whole monastery.
-QUEST_LV_0300_20160310_001283	You've done it!{nl}The black wall disappeared..{nl}
-QUEST_LV_0300_20160310_001284	Thank you for saving the monastery.{nl}Ah, is the superior father alright?{nl}
-QUEST_LV_0300_20160310_001285	..So that is it. {nl}Your religious belief is strong as I imagined.
+QUEST_LV_0300_20160310_001281	Even if the world collapses, we have hope like you...{nl}You give us the power to endure.{nl}
+QUEST_LV_0300_20160310_001282	I don't want to imagine what would have happened without you...{nl}On behalf of the whole monastery, I very much appreciate your help.
```

Comment:
"On behalf of the whole monastery, we greatly appreciate your help."  I feel like it should be "we" instead of "I," as he's talking on the behalf of the group.